### DISCUSSION TITLE
[Cloudflare preset] Wrangler dev/publish hashes (renames) static file such as favicon.ico

### DISCUSSION THREAD

**Discussion by AlexDicy** [nitrojs/nitro#188] [Q&A]
I'm working on a Nuxt 3 project, and I put static assets inside the `public` directory.

It works fine when using `nuxt dev` and `nuxt build` (the files are not renamed/hashed inside the .output directory, which is the wanted behavior).

The issue is when I try to deploy the website using wrangler publish the assets are uploaded fine but renamed/hashed.

For example `/public/favicon.ico` becomes `.output/public/favicon.ico` and is uploaded to Cloudflare Workers as `favicon.12b8671dff.ico` which breaks it.

Have you experienced anything similar? How did you handle it?

**Comment by ShaggyTech** [comment]
I'm running into the same problem.  Other images are hashed but still show correctly, favicon refuses to load due to the hashed name
